recipes/glib/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set major_minor = "2.48" %}
+{% set version = major_minor + ".0" %}
+{% set sha256 = "744be6931ca914c68af98dc38ff6b0cf8381d65e335060faddfbf04c17147c34" %}
```

Comment:
But it's used in only one place. I understand the purpose of `{{ version }}` because it's used in several places, but this one seems superfluous to me.